test/fail_compilation/fail16997.d

```diff
@@ -1,5 +1,5 @@
 /*
-REQUIRED_ARGS: -de
+REQUIRED_ARGS: -de -revert=intpromote
```

Comment:
Here you're using revert to get a deprecation message, but the changelog says deprecation is the default, which is right?

Reply:
I am focusing on getting the test to pass here.